release-notes.md

```diff
@@ -1,5 +1,21 @@
 # Release notes
 
+## April 16, 2019
+
+Project Griffon is now available for beta access!
+
+Project Griffon is a new, innovative product from Adobe to help you easily inspect, validate, debug data collection and experiences for your mobile app. Made for app product managers and developers, this product will do the heavy lifting of getting your app implementation right, while you focus on creating engaging experiences.
+
+Project Griffon is in beta. Use of this beta product requires acceptance of terms outlined on [https://griffon.adobe.com](https://griffon.adobe.com).
```

Comment:
Please change text to:

"Project Griffon is in beta, and the use of this beta release requires you to accept the terms outlined in <link>." 

(I think this sentence is also in the docs, which I'll fix.)